PYME/recipes/processing.py

```diff
@@ -2848,3 +2848,36 @@ def run(self, input_name):
         series_epers.mdh['Camera.ADOffset'] = 0
 
         return series_epers
+    
+
+@register_module('Rotate180')
+class Rotate180(Filter):
+    """
+    Rotate the image 180 degrees
+
+    Parameters
+    ----------
+    input_name : PYME.IO.ImageStack
+    output_name : PYME.IO.ImageStack
+    """
```

Comment:
```suggestion
    """
    
    dimensionality = Enum('XY', desc='Which image dimensions should the filter be applied to?')
```
Because the code in `applyFilter` expects a 2D array, we have to redefine the dimensionality trait to ensure it doesn't get 3D (or 4D) data.

Reply:
If we wanted to be super safe, we could also add

`assert(data.ndim ==2)` to the beginning of `apply_filter`. This would only really be helpful in terms of giving a useful error message if somehow it did end up being called with 3D data.